arm/.global/global.module.tests.ps1

```diff
@@ -764,7 +764,7 @@ Describe 'Deployment template tests' -Tag Template {
             $outputs | Should -Contain 'resourceId'
         }
 
-        It "[<moduleFolderName>] parameters' description shoud start with a one word category followed by a dot, a space and the actual description text." -TestCases $deploymentFolderTestCases {
+        It "[<moduleFolderName>] parameters' description shoud start with a one word category followed by a dot, a space and the actual description text ending with a dot." -TestCases $deploymentFolderTestCases {
```

Comment:
Should the same be tested for the outputs? You updated those too

Reply:
Was thinking about the same thing, but that would be an additional test. What do you think?

Reply:
I'd suggest to add one. More tests is not a bad thing - even if people that cloned the current repo may hate us for it. 
Test-wise it should be very simple as it's almost a 1:1 copy of the current test - just pointing to a different metadata block

Reply:
done